### PR TITLE
Add opusfile.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

File: opusfile/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,18 @@
+pkgname=opusfile
+pkgver=0.10
+pkgrel=1
+url="https://opus-codec.org/"
+source=("git://github.com/xiph/opusfile")
```
Comment (devnoname120) [inline opusfile/VITABUILD]: `#tag=v$pkgver`

File: opusfile/VITABUILD [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+sha256sums=('SKIP')
+
+build() {
+  cd $pkgname
+  ./autogen.sh
+  ./configure --host=arm-vita-eabi --prefix=$prefix --disable-shared --enable-static --enable-fixed-point
+  make
```
Comment (devnoname120) [inline opusfile/VITABUILD]: `make -j$(nproc)`

File: opusfile/VITABUILD [hunk 3]
```diff
@@ -0,0 +1,18 @@
+pkgname=opusfile
+pkgver=0.10
+pkgrel=1
+url="https://opus-codec.org/"
+source=("https://github.com/xiph/$pkgname/archive/v$pkgver.tar.gz")
+sha256sums=('bd9c246cf18d27e9a0815e432731d82f0978717fe2dc2b1e1dce09c184132239')
+
```
Comment (devnoname120) [inline opusfile/VITABUILD]: `depends=('ogg' 'opus')`

**Comment by devnoname120** [comment]
@d3m3vilurr any ideas how to fix this? https://travis-ci.org/vitasdk/packages/builds/636640065#L31599-L31600

**Comment by d3m3vilurr** [comment]
vita-makepkg command was lost the `depends` information before https://github.com/vitasdk/vita-makepkg/commit/f99b7c47a2.
it could be a problem of building environ or we should recheck this patch.

**Pull request merged**